packages/components/src/Form/Inputs/InputFilters/InputFilters.tsx

```diff
@@ -164,6 +182,7 @@ const InputFiltersLayout: FC<InputFiltersProps> = ({
         <Select
           autoResize
```

Comment:
I think `openOnFocus` would be a nice feature for the user – they'll see the options list open when they click anywhere in the input.
```suggestion
          autoResize
          openOnFocus
```